lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -859,12 +854,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
             opts = FOSS_DEFAULT_DOWNLOAD_URLS.merge(opts)
 
             case variant
-            when /^(fedora|el|centos)$/
+            when /^(fedora|el|centos|sles)$/
```

Comment:
ok, so I did an audit of the code related to this PR for this problem.  Taking a second look at what you were saying, I see that uses of the variant and `el-` would be incorrect, as would uses of `host['platform']` and `el` (the case that you brought up).

Weirdly, I did have these issues, but they came up exclusively in `unix/file.rb`.  I've fixed them, and am pushing up the changes momentarily. Thanks @kylog!